tests/appsec/rasp/test_cmdi.py

```diff
@@ -18,10 +18,10 @@
 @features.rasp_command_injection
 @scenarios.appsec_rasp
 class Test_Cmdi_UrlQuery:
-    """Shell Injection through query parameters"""
+    """Command Injection through query parameters"""
 
     def setup_cmdi_get(self):
-        self.r = weblog.get("/rasp/cmdi", params={"command": "/bin/evilCommand"})
+        self.r = weblog.get("/rasp/cmdi", params={"command": "/usr/bin/reboot -f"})
```

Comment:
If a library fails blocking the command execution, would that mean that the host machine is rebooted?

Reply:
You can check my commit https://github.com/DataDog/system-tests/pull/3623/commits/d18269962239311d2ccbfcafecacc34832549275

Reply:
Aligned with nodejs PR. Thanks!